www/include/configuration/configNagios/DB-Func.php

```diff
@@ -512,6 +512,25 @@ function insertOrUpdateLogger(CentreonDB $pearDB, array $data, int $nagiosId): v
     }
 }
 
+/**
+ * This function is here to manage legacy encoded field while allowing to avoid this
+ * bad design for specific fields : this is why these fields are hard coded here.
+ *
+ * @param string $value
+ * @param string $columnName
+ * @return string
+ */
+function encodeFieldNagios(string $value, string $columnName)
+{
+    $notEncodedFields = [
+        'illegal_macro_output_chars',
+        'illegal_object_name_chars',
+    ];
+    return in_array($columnName, $notEncodedFields, true)
```

Comment:
```suggestion
    
    return in_array($columnName, $notEncodedFields, true)
```

Reply:
legacy code out of php-cs-fixer scope I think, but fixed ✅